discovery/seata-discovery-etcd3/pom.xml

```diff
@@ -49,12 +43,10 @@
         <dependency>
             <groupId>io.etcd</groupId>
             <artifactId>jetcd-launcher</artifactId>
-            <scope>test</scope>
```

Comment:
jetcd-launcher is only used for testing.

Reply:
See the bom.xml

Reply:
> See the bom.xml

got it.